app/include/zmk/ble.h

```diff
@@ -36,6 +36,8 @@ char *zmk_ble_active_profile_name(void);
 
 int zmk_ble_unpair_all(void);
 
+void zmk_ble_set_device_name(char *name);
```

Comment:
This should really return `int` and propagate any errno returned by `bt_set_name`.

Reply:
Addressed. Also returned any errors from `update_advertising`